tests/integration/conftest.py

```diff
@@ -42,6 +42,8 @@
 
 COTS_CONTRIBUTOR_ID = "rt.tchoutchou"
 PIV_CONTRIBUTOR_ID = "rt.piv"
+PIV_EXCHANGE_NAME = "piv"
+PIV_QUEUE_NAME = "piv_kirin"
```

Comment:
To match naming decided:
```suggestion
PIV_EXCHANGE_NAME = "piv_schedule"
PIV_QUEUE_NAME = "piv_schedule_kirin"
```

Reply:
Done in 4f5b6898bd4488427323e888838cbff87ae0a97c.